python/consumer/kafka_consumer_config.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+KAFKA_PREFIX = "KAFKA_"
+STRIMZI_PREFIX = "STRIMZI"
+
+# Strimzi environment variables declaration
+TOPIC = "STRIMZI_TOPIC"
+MESSAGE_COUNT = "STRIMZI_MESSAGE_COUNT"
+
+# default values for Strimzi environment variables
+DEFAULT_TOPIC = "my-topic"
+DEFAULT_MESSAGE_COUNT = 10
+
+# default values for Kafka environmental variables in a dictionary
+DEFAULT_PROPERTIES = {
+    'bootstrap.servers': 'localhost:9092',
+    'group.id': 'my-kafka-consumer',
+    'auto.offset.reset': 'latest',
+}
+
+
+def convertEnvVarToPropertyKey(e):
+    return e[e.index('_') + 1:].lower()
+
+
+class KafkaConsumerConfig:
+    def __init__(self, topic, message_count, properties):
+        self.topic = topic
+        self.message_count = message_count
```

Comment:
It looks like we don't use this anywhere

Reply:
I had planned on introducing some more logic like ` while receivedMsgs < config.message_count: .....` to the consumer but for now I'll keep it simple and remove message_count